chapters/hpmor-chapter-064.tex

```diff
@@ -270,44 +270,44 @@ \chapter{Omake Files IV, Alternate Parallels}
 and the good heart of my good friend Horatio.\\
 Or else, if thou hast true title to this belov’d form, tell me:\\
 What drawing did I present to Hamlet King,\\
-when six years old and scarce out of my sling?\\
+when six years old and scarce out of my sling?
```

Comment:
A bunch of line breaks removed here; why? Surely this will change the LaTeX output?

Reply:
Sorry, my bad, I thought it would have no effect, but it does, so removed and moved the corresponding fix to the e-book gen script. Thank you so much for pointing this out. Should have double checked myself prior to committing.